### PR TITLE
added xcms for R

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
@boegel any remarks?

**Comment by boegel** [comment]
Jenkins: test this pleae

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/r/R/R-3.1.1-intel-2014b.eb [hunk 1]
```diff
@@ -222,6 +222,7 @@ exts_list = [
     ('GenomicAlignments', '1.0.4', bioconductor_options),
     ('ShortRead', '1.22.0', bioconductor_options),
     ('graph', '1.42.0', bioconductor_options),
+    ('doMC', '1.3.3', bioconductor_options),
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.1.1-intel-2014b.eb]: we won't start updating old R easyconfigs; please flesh this out as a stand-alone easyconfig?

Comment (JensTimmerman) [inline easybuild/easyconfigs/r/R/R-3.1.1-intel-2014b.eb]: xcms does depend on doMC, so can't just flesh this out, but I Can change the entire pull request to target a newer R...

Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.1.1-intel-2014b.eb]: you can create a stand-alone easyconfig for doMC, no?

**Comment by migueldiascosta** [comment]
@JensTimmerman, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**